crypto_misc.bib

```diff
@@ -1,5 +1,221 @@
+  title        = {{HQC}},
+  institution  = {{N}ational {I}nstitute of {S}tandards and {T}echnology},
+  year         = 2019,
+  note         = {available at \url{https://csrc.nist.gov/projects/post-quantum-cryptography/round-2-submissions}},
+}
+
+@techreport{NISTPQC-R2:LAC19,
+  author       = {Xianhui Lu and Yamin Liu and Dingding Jia and Haiyang Xue and Jingnan He and Zhenfei Zhang and Zhe Liu and Hao Yang and Bao Li and Kunpeng Wang},
+  title        = {{LAC}},
+  institution  = {{N}ational {I}nstitute of {S}tandards and {T}echnology},
+  year         = 2019,
+  note         = {available at \url{https://csrc.nist.gov/projects/post-quantum-cryptography/round-2-submissions}},
+}
+
+@techreport{NISTPQC-R2:LEDAcrypt19,
+  author       = {Marco Baldi and Alessandro Barenghi and Franco Chiaraluce and Gerardo Pelosi and Paolo Santini},
+  title        = {{LEDAcrypt}},
+  institution  = {{N}ational {I}nstitute of {S}tandards and {T}echnology},
+  year         = 2019,
+  note         = {available at \url{https://csrc.nist.gov/projects/post-quantum-cryptography/round-2-submissions}},
+}
+
+@techreport{NISTPQC-R2:NewHope19,
+  author       = {Thomas Poppelmann and Erdem Alkim and Roberto Avanzi and Joppe Bos and L{\'e}o Ducas and Antonio de la Piedra and Peter Schwabe and Douglas Stebila and Martin R. Albrecht and Emmanuela Orsini and Valery Osheter and Kenneth G. Paterson and Guy Peer and Nigel P. Smart},
+  title        = {{NewHope}},
+  institution  = {{N}ational {I}nstitute of {S}tandards and {T}echnology},
+  year         = 2019,
+  note         = {available at \url{https://csrc.nist.gov/projects/post-quantum-cryptography/round-2-submissions}},
+}
+
+@techreport{NISTPQC-R2:NTRU19,
+  author       = {Zhenfei Zhang and Cong Chen and Jeffrey Hoffstein and William Whyte and John M. Schanck and Andreas Hulsing and Joost Rijneveld and Peter Schwabe and Oussama Danba},
+  title        = {{NTRUEncrypt}},
+  institution  = {{N}ational {I}nstitute of {S}tandards and {T}echnology},
+  year         = 2019,
+  note         = {available at \url{https://csrc.nist.gov/projects/post-quantum-cryptography/round-2-submissions}},
+}
+
+@techreport{NISTPQC-R2:NTRUPrime19,
+  author       = {Daniel J. Bernstein and Chitchanok Chuengsatiansup and Tanja Lange and Christine {van Vredendaal}},
+  title        = {{NTRU Prime}},
+  institution  = {{N}ational {I}nstitute of {S}tandards and {T}echnology},
+  year         = 2019,
+  note         = {available at \url{https://csrc.nist.gov/projects/post-quantum-cryptography/round-2-submissions}},
+}
+
+@techreport{NISTPQC-R2:NTS-KEM19,
+  author       = {Martin Albrecht and Carlos Cid and Kenneth G. Paterson and Cen Jung Tjhai and Martin Tomlinson},
+  title        = {{NTS-KEM}},
+  institution  = {{N}ational {I}nstitute of {S}tandards and {T}echnology},
+  year         = 2019,
+  note         = {available at \url{https://csrc.nist.gov/projects/post-quantum-cryptography/round-2-submissions}},
+}
+
+ROLLO
```

Comment:
```suggestion 
```

Reply:
I added these changes separately to cryptobib, including the change suggested above.